internal/cmd/diag.go

```diff
@@ -48,13 +47,21 @@ func ExecInShimHost(ctx context.Context, req *shimdiag.ExecProcessRequest) (int,
 	if len(req.Args) > 1 {
 		cmdArgsWithoutName = req.Args[1:]
 	}
+	np, err := NewNpipeIO(ctx, req.Stdin, req.Stdout, req.Stderr, req.Terminal)
+	if err != nil {
+		return 0, err
```

Comment:
Return -1 here?

Reply:
Ehh we don't for `ExecInUvm` in this case either, I usually return the error case if err != nil but this is just a nit anyways.